test/local/token_expiry_tests.js

```diff
@@ -6,7 +6,7 @@ var test = require('../ptaptest')
 var TestServer = require('../test_server')
 var Client = require('../client')
 
-process.env.PASSWORD_CHANGE_TOKEN_TTL = '-1'
+process.env.PASSWORD_CHANGE_TOKEN_TTL = '1'
```

Comment:
> I personally would rather have a way to set the config value using something else rather than process.env. 

Agreed.  Are these mostly used to pass config values through to the subprocess spawned by `TestServer`?  Perhaps we can refactor it to take the required config directly as arguments to its constructor instead of relying on them being in the environment.